ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -1442,12 +1442,10 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      E2E_RUN_TAGS: '@ota'
       TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&
-      TEST_SCENARIOS: OTA
       TEST_TIMEOUT: "75"
```

Comment:
Perhaps also define `TEST_FILTERS` and `TEST_TIMEOUT` in the chain to simplify the configs in the job level ?

Reply:
At least in this PR, 1 setting in the chain can replace 7 settings in the jobs.

Reply:
updated